docs/content/3.application-side/5.guest-mode.md

```diff
@@ -15,6 +15,7 @@ This behavior is useful for login pages that you don't want to be visitable by l
 Briefly summarized, you can enable guest mode on a page by passing the following configuration:
 ```ts
 definePageMeta({
+  middleware: 'auth',
```

Comment:
Maybe we add a comment behind it indicating that this is not required if the `globalAppMiddleware` is enabled?

```suggestion
  middleware: 'auth', // Only required if globalAppMiddleware is disabled
```

Reply:
Thanks for the suggestion, took it in the commit 🙏